src/test/java/io/logz/log4j2/Log4j2AppenderTest.java

```diff
@@ -212,36 +213,8 @@ public void testTokenAndLogzioUrlFromSystemEnvironment() {
         LogRequest logRequest = mockListener.assertLogReceivedByMessage(message1);
         mockListener.assertLogReceivedIs(logRequest, token, type, loggerName, Level.INFO.name());
     }
-
-    private Logger createLogger(String token, String type, String loggerName, Integer drainTimeout,
-                                  boolean addHostname, String additionalFields, boolean compressRequests) {
-        logger.info("Creating logger {}. token={}, type={}, drainTimeout={}, addHostname={}, additionalFields={}",
-                loggerName, token, type, drainTimeout, addHostname, additionalFields);
-        Logger log4j2Logger =  LogManager.getLogger(loggerName);
-        LogzioAppender.Builder logzioLog4j2AppenderBuilder = LogzioAppender.newBuilder();
-        logzioLog4j2AppenderBuilder.setLogzioToken(token);
-        logzioLog4j2AppenderBuilder.setLogzioType(type);
-        logzioLog4j2AppenderBuilder.setDebug(true);
-        logzioLog4j2AppenderBuilder.setLogzioUrl("http://" + mockListener.getHost() + ":" + mockListener.getPort());
-        logzioLog4j2AppenderBuilder.setAddHostname(addHostname);
-        logzioLog4j2AppenderBuilder.setCompressRequests(compressRequests);
-
-        if (drainTimeout != null) {
-            logzioLog4j2AppenderBuilder.setDrainTimeoutSec(drainTimeout);
-        }
-
-        if (additionalFields != null) {
-            logzioLog4j2AppenderBuilder.setAdditionalFields(additionalFields);
-        }
-
-        LogzioAppender appender = logzioLog4j2AppenderBuilder.build();
-        appender.start();
-        assertThat(appender.isStarted()).isTrue();
-        ((org.apache.logging.log4j.core.Logger) log4j2Logger).addAppender(appender);
-        ((org.apache.logging.log4j.core.Logger) log4j2Logger).setAdditive(false);
-
-        return log4j2Logger;
-    }
+    abstract Logger createLogger(String token, String type, String loggerName, Integer drainTimeout,
```

Comment:
lets use parametrized list - https://github.com/junit-team/junit4/wiki/parameterized-tests
and remove the extends

Reply:
But he creates a different logger